PubSub/src/Topic.php

```diff
@@ -518,6 +518,8 @@ public function publishBatch(array $messages, array $options = [])
      *     @type bool $debugOutput Whether or not to output debug information.
      *           Please note debug output currently only applies in CLI based
      *           applications. **Defaults to** `false`.
+     *     @type resource $debugOutputResource A resource to output debug output
+     *           to. **Defaults to** `php://stderr`.
```

Comment:
The value actually defaults to `fopen('php://stderr', 'w')` and not exactly `'php://stderr'`.

```suggestion
     *     @type resource $debugOutputResource A resource to output debug output
     *           to. **Defaults to** Resource for `php://stderr`.
```